linera-core/src/client/mod.rs

```diff
@@ -922,9 +922,7 @@ where
             .await?
             .info;
         let epoch = info.epoch;
-        let committees = info
-            .requested_committees
-            .ok_or(LocalNodeError::InvalidChainInfoResponse)?;
+        let committees = info.requested_committees.unwrap_or_else(BTreeMap::new);
```

Comment:
I'm not sure I agree this is a good change. I understand it's necessary for the fix but:
- it's not possible for the protocol to execute correctly with empty committee
- if we requested it then it shouldn't be empty.

Reply:
On the call-site we only pattern match on the first element of the tuple and interpret the `None` as a chain not-active. Maybe we can return `Ok(None)` entirely.

Reply:
This also triggers a bunch of other changes, and I was trying to keep this PR small.

I'll look into cleaning up all those committee getters a bit; if it's not too big I'll add it to this PR.